README.md

```diff
@@ -46,7 +49,7 @@ pytest
 
 ## Credits
 
-- [Numan Ijaz](NumanIjaz)
```

Comment:
If you like, you can add your name here as well (because the contributors link doesn't work in pypi.org)

Reply:
Thanks. Done.